sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -716,25 +715,6 @@ public void recognizePiiEntitiesEmoji(HttpClient httpClient, TextAnalyticsServic
         );
     }
 
-    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
-    @MethodSource("com.azure.ai.textanalytics.TestUtils#getTestParameters")
-    @Disabled("https://github.com/Azure/azure-sdk-for-java/issues/22257")
```

Comment:
Just curious on follow up for https://github.com/Azure/azure-sdk-for-java/issues/22257

Reply:
I could probably Moving StringIndexType to Core in next month.

Reply:
Keep me posted on this one. I have an issue in the backlog for doing this for .NET too: https://github.com/Azure/azure-sdk-for-net/issues/21370